### PR TITLE
feat: add operator as a sub-organisation

### PR DESCRIPTION
**Pull request by DeannaSharma**
Designs show an email address field, however we do not need this field. Designs to be updated  next week.

### CODE REVIEW

File: site/pages/admin/__snapshots__/user-management.test.tsx.snap [hunk 1]
```diff
@@ -107,6 +107,16 @@ exports[`userManagement > should render correctly when there are no inputs 1`] =
...
       </a>
+      <a
+        className="govuk-button--secondary govuk-button mt-5"
+        href="/admin/add-operator"
+        onClick={[Function]}
+        onMouseEnter={[Function]}
+        onTouchStart={[Function]}
+        role="button"
+      >
+        Add new Operator
```
Comment (alainajaffer) [inline site/pages/admin/__snapshots__/user-management.test.tsx.snap:118]:
```suggestion
        Add new operator
```
I feel like this should be lower case to be consistent with Add new user

File: site/schemas/organisation.schema.ts [hunk 2]
```diff
@@ -33,3 +33,19 @@ export const areaCodeSchema = z.string().transform((item) => ({
...
+
+export type SubOrganisation = z.infer<typeof subOrganisationSchema>;
+
+export const subOrganisationsSchema = z.array(
+    z.object({
+        name: z.string(),
+        PK: z.string().optional(),
+        nocCodes: z.array(z.string()),
+    }),
+);
```
Comment (alainajaffer) [inline site/schemas/organisation.schema.ts:51]:
```suggestion
export const subOrganisationsSchema = z.array(subOrganisationSchema);
```

File: site/pages/admin/__snapshots__/user-management.test.tsx.snap (hunk 1 above)
Comment (alainajaffer) [reply site/pages/admin/__snapshots__/user-management.test.tsx.snap:118]: as discussed - sticking to designs

File: site/schemas/organisation.schema.ts (hunk 2 above)
Comment (DeannaSharma) [reply site/schemas/organisation.schema.ts]: done

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**